portfolio-project-matching-app/pages/myApplications.js

```diff
@@ -221,7 +221,7 @@ const myApplications = () => {
                     </div>
                     <ApplicationList apps={isOutgoing ? visibleOutApplications : visibleInApplications} isOutgoing={isOutgoing} />
                 </div>
-                <div className='sticky top-4 m-2 p-2 col-span-1 h-auto defaultBorder bg-white shadow-2xl'>
+                <div className='sticky top-4 m-2 p-2 col-span-1 h-auto max-h-96 defaultBorder bg-white shadow-2xl'>
```

Comment:
Getting things consistent with `browseProjects`. Might need to update the max height in case the number of filter buttons grows.